src/components/AuthContainer.js

```diff
@@ -14,8 +15,11 @@ import TermsOfService from 'src/pages/TermsOfService'
 const AuthContainer = ({ children }) => {
   const { name, public: isPublic } = useRoute()
   const { isSignedIn, registrationStatus, acceptedTos, profile } = useStore(authStore)
+  const auth = useAuth()
   const authspinner = () => h(centeredSpinner, { style: { position: 'fixed' } })
 
+  useOnMount(() => authStore.update(_.set(['authContext'], auth)))
```

Comment:
This is grabbing the `AuthContext` out of React context and setting it in our state atom. I do this so I can access it in `src/libs/auth.js` outside of a react component.

Alternatively: we could update every component to call `useAuth()` and pass the `AuthContext` around as needed. That change would touch more files though. See usage examples in https://github.com/authts/react-oidc-context

Reply:
Good call on limiting the scope. We can iterate toward the right approach.

Reply:
Can this be removed now that you have `AuthStoreSetter`?